docs/src/multi-physics_coupling.md

```diff
@@ -0,0 +1,37 @@
+# [Multi-physics coupling](@id multi-physics-coupling)
+A complex simulation can consist of different spatial domains in which
+different equations are being solved, different numerical methods being used
+or the grid structure is different.
+One example would be a fluid in a tank and an extended hot plate attached to it.
+We would then like to solve the Navier-Stokes equations in the fluid domain
+and the heat conduction equations in the plate.
+The coupling would happen at the interface through the exchange of thermal energy.
+
+
+## Converter Coupling
+We can have the case where the two systems do not share any variables, but
+share some of the physics.
+Here, the same physics is just represented in a different form and with
+different variables.
+This is the case for a fluid system on one side and a Vlasov system on the other.
```

Comment:
```suggestion
This is the case, for instance assuming two domains, if there is a fluid system in one domain
and a Vlasov system in the other domain.
```
This is a very specific example and not everyone will necessarily know what a Vlasov system is. Also, the given example elixir couples four different advective equations where the variables "match". It might be worth mentioning more explicitly somewhere that these converter routines need to be hand-rolled by the user within an elixir.

Reply:
Thanks, I implemented these changes in the coupling PR.